pkgs/development/compilers/ghc/with-packages.nix

```diff
@@ -44,6 +47,7 @@ buildEnv {
       rm -f $out/bin/$prg
       makeWrapper ${ghc}/bin/$prg $out/bin/$prg         \
         --add-flags '"-B$NIX_GHC_LIBDIR"'               \
+        ${optionalString exposeGHC "--add-flags '-package ghc'"} \
```

Comment:
I'm not sure what the addition of this flag is meant to accomplish. What is the use-case for this change?